InternxtDesktop/Views/Settings/SettingsView.swift

```diff
@@ -51,6 +53,28 @@ struct SettingsView: View {
                 .frame(maxWidth: .infinity, maxHeight: .infinity)
                 .background(Color.Gray1.opacity(0.8))
             }
+
+            // stop ongoing backup dialog
+            if showStopBackupDialog {
+                VStack {
+                    StopBackupDialog {
```

Comment:
use a named property here, otherwise I have no idea what this closure is doing, something like:

```
StopBackupDialog(onClose: {
    showStopBackupDialog = false
})
```